spec.md

```diff
@@ -187,10 +187,40 @@ A web-friendly display name is camel-cased with spaces and punctuation supported
 | metadata | JSON object | An opaque object of metadata for a service plan. Controller treats this as a blob. Note that there are [conventions](https://docs.cloudfoundry.org/services/catalog-metadata.html) in existing brokers and controllers for fields that aid in the display of catalog data. |
 | free | boolean | When false, service instances of this plan have a cost. The default is true. |
 | bindable | boolean | Specifies whether service instances of the service plan can be bound to applications. This field is OPTIONAL. If specified, this takes precedence over the `bindable` attribute of the service. If not specified, the default is derived from the service. |
+| [schemas](#schema-object) | object | Schema definitions for service instances and bindings for the plan. |
 
 
 \* Fields with an asterisk are REQUIRED.
 
+##### Schema Object
+
+| Response field | Type | Description |
+| --- | --- | --- |
+| [service_instance](#service-instances-object) | object | The schema definitions for all actions for a service instance. |
+| [service_binding](#service-bindings-object) | object | The schema definitions for all actions for a service binding. Used only if the service is bindable. |
```

Comment:
Suggestion for description:

"The schema definitions for a service binding. Used only if the service is bindable."